frontend/src/views/FarmInformation/FarmInformation.tsx

```diff
@@ -92,10 +121,11 @@ export default function FarmInformation() {
 
     apiCache.callEndpoint('api/regions/').then((response) => {
       const { data } = response;
-      const regions: { value: number; label: string }[] = (
-        data as { id: number; name: string }[]
-      ).map((row) => ({ value: row.id, label: row.name }));
-      setRegionOptions(regions);
+      const regions = (data as { id: string; name: string }[]).map((row) => ({
```

Comment:
The ids are not strings. They are numbers. Javascript doesn't actually care, but this isn't a correct cast.